--- jsonata/jsonata_test.go
+++ jsonata/jsonata_test.go
@@ -3,7 +3,6 @@
 import (
	"encoding/json"
-	"fmt"
	"testing"

	"github.com/arran4/lookup"
	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
@@ -106,4 +105,2 @@

-				if fmt.Sprintf("%v", tt.want) != fmt.Sprintf("%v", raw) {
-					t.Errorf("Expected %#v, got %#v", tt.want, raw)
-				}
+				require.Equal(t, tt.want, raw)
			})
